### DISCUSSION TITLE
Parameter Randomization Question

### DISCUSSION THREAD

**Discussion by markrhysmitchell** [kernicPanel/bitwig-randomizer#14] [General]
I wrote to Bitwig a couple of years ago requesting this feature so I have to say your extension is a dream come true for me. In the past I've been using Bidule to achieve this but this is so much nicer.

It doesn't seem to work globally though across all plugin devices. For example, all the d16 VSTs randomize perfectly, but the u-he ones don't move at all. Is this a limitation of the extension or a possible bug?

Thanks again!

EDIT: Apologies, this was based on a misunderstanding of how the randomizer works. It can't randomize plugin parameters directly like in Bidule - they must be exposed to a Preset or Device Page.

**Comment by kernicPanel** [comment]
Hi,
You're right, for now, only remote controls are available.
I tried to access plugin parameters directly, without luck.
I also requested it to Bitwig.
Maybe the API would evolve in this direction in the future ?

Thanks for your kind words, and have fun with randomness.
